### DISCUSSION TITLE
Working Group Catchup | June 7th

### DISCUSSION THREAD

**Discussion by PoojaJManjooran** [Sunbird-inQuiry/Community#89] [Announcements]
Hi Folks,
We are planning a working group catchup next week.

Date : 7th June, Wednesday
Time : 5 pm
Duration : 1 hour

**Join Zoom Meeting**
https://ekstep-org.zoom.us/j/86021689750?pwd=ZE54UmlsUE1TNUZkMmZITWw0aUpJdz09

Meeting ID: 860 2168 9750
Passcode: 907578

- In this call we shall discuss the inQuiry roadmap (for the next 6 months).

- I would request all the working group members to post items (as comments on this thread) that you would like to discuss during the catchup.

_**The suggested team structure of the working group is to have a maximum of three members from one organisation -  Preferably 1 Product lead, 1 Tech lead and one more if required (could be an architect or a key developer).**_ 

@Prateek-slokam @aks30 @kiranharidas187  @devendra-tarento @santhosh-tarento @sajeshkayyath @kameshbhr @Krishnaj20 @vikram

**Comment by PoojaJManjooran** [comment]
Hi Folks!
For the working group catchup tomorrow, I would like request you to post  in this thread, anything that you would like to particularly discuss.

Also, we shall plan to have the suggested team structure as mention in the above thread.

@Prateek-slokam @aks30 @kiranharidas187 @devendra-tarento @santhosh-tarento @sajeshkayyath @kameshbhr @Krishnaj20 @vikram

**Comment by sajeshkayyath** [comment]
Hi Folks!

Thanks for joining the call today. Below are the notes from the meeting for wider audience who might have missed the call.

- Inquiry team is working towards the release of 6.0.0 as per the schedule and plan items mentioned [here](https://github.com/orgs/Sunbird-inQuiry/discussions/79)
- SikshaLokam is working towards making the QuML compliance changes, feature to capture incremental review/rejection comments.
- Tarento is working towards Blue Print, Dynamic Question Set and Exam capabilities

Some action points are as follows
- Create a space for the contributors to share their roadmap items
- Information on the release / regression testing process and the timeline expectations to catch the release to be shared with the contributors
- Arrange a call for sharing the QuML compliance changes with the contributors. The previous thread is [here](https://github.com/orgs/Sunbird-inQuiry/discussions/83) on the same subject.
- Tarento to share the new feature related documents that it is building in inQuiry for the benefit of wider audience
- Share the recordings for the future sessions

@PoojaJManjooran  @Prateek-slokam @aks30 @kiranharidas187 @devendra-tarento @santhosh-tarento @sajeshkayyath @kameshbhr @Krishnaj20 @vikram @anup1901 @SaipradeepR @thillaibirla

**Reply by PoojaJManjooran** [reply]
Hi folks,
As discussed in the last Working Group catchup, we will be having a call to discuss the up-gradation of Sunbird inQuiry to the latest version of theQuML Spec.

Join Zoom Meeting
[https://ekstep-org.zoom.us/j/83741030372?pwd=SGg3dzk4bDFKQzJiandqUi93WTRhZz09](https://www.google.com/url?q=https://ekstep-org.zoom.us/j/83741030372?pwd%3DSGg3dzk4bDFKQzJiandqUi93WTRhZz09&sa=D&source=calendar&ust=1687243805025203&usg=AOvVaw1YxVZHm5010WqxFMD7LXFC)

**Date : 19th June, Monday**
Time : 5pm
Meeting ID: 837 4103 0372
Passcode: 691854

Note : This will be a critical change for the inquiry adopters and hence I suggest all the adopters join this call or add the relevant folks from your team as well.

@Prateek-slokam @aks30 @kiranharidas187 @devendra-tarento @santhosh-tarento @sajeshkayyath @kameshbhr @Krishnaj20 @anup1901 @SaipradeepR @thillaibirla

**Reply by PoojaJManjooran** [reply]
Sharing the recording of the call [here](https://ekstep-org.zoom.us/rec/play/eBD2kARvPamAZY20ukKvMc-0Wk4O_C9KYKLCosYb4y_BlvsyYUjierS1-qSA0iMOdiqhHN9nGLdVaI_L.WL0_4l0DcFzjSZPj?canPlayFromShare=true&from=my_recording&continueMode=true&componentName=rec-play&originRequestUrl=https%3A%2F%2Fekstep-org.zoom.us%2Frec%2Fshare%2FmfusvXrq8v2LqnwMu2Ar0pPGPbXWAIazqU1MyCME7dwChRRsqqorQWXT3gSznbz7.RF1Xzk9aET6XOqPX)

Attaching the confluence doc with the Release 6.0.0 changes [here](https://project-sunbird.atlassian.net/l/cp/4uMKq8sG)

@Prateek-slokam @aks30 @kiranharidas187 @devendra-tarento @santhosh-tarento @sajeshkayyath @kameshbhr @anup1901 @SaipradeepR @thillaibirla @ramkumar-pacewisdom
